modules/psap-special-resource-operator-using-configmaps.adoc

```diff
@@ -0,0 +1,310 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-special-resource-operator.adoc
+
+[id="deploy-simple-kmod-using-configmap-chart"]
+= Building and running the simple-kmod SpecialResource by using a ConfigMap
+
+In this example, the simple-kmod kernel module is used to show how the loading of a kernel module can be managed by SRO, by storing Helm chart templates in a ConfigMap.
```

Comment:
I think this could be worded in a less awkward / passive way.

```suggestion
In this example, the simple-kmod kernel module is used to show how SRO can manage a driver container which is defined in Helm chart templates stored in a ConfigMap.
```

Reply:
fixed!